core/src/main/java/com/airbnb/aerosolve/core/function/Point.java

```diff
@@ -0,0 +1,93 @@
+package com.airbnb.aerosolve.core.function;
+
+import com.airbnb.aerosolve.core.FunctionForm;
+import com.airbnb.aerosolve.core.ModelRecord;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Point implements Function {
+  private float weight;
+
+  public Point(float weight) {
+    this.weight = weight;
+  }
+
+  public Point(ModelRecord record) {
+    List<Double> weightVec = record.getWeightVector();
+    weight = weightVec.get(0).floatValue();
+  }
+
+  @Override
+  public Function aggregate(Iterable<Function> functions, float scale, int numBins) {
+    float aggWeight = 0;
+    for (Function fun: functions) {
+      Point point = (Point) fun;
+      aggWeight += scale * point.weight;
+    }
+    return new Point(aggWeight);
+  }
+
+  @Override
+  public float evaluate(float... x) {
```

Comment:
for string feature, we assume `x=1`, probably we should return `weight * x` so that using `Point` to represent features in AdditiveModel would be make this additive model equivalent to conventional linear model.